app/components/ui/Divider.tsx

```diff
@@ -0,0 +1,7 @@
+import { Box } from "@mui/material"
+
+export const Divider = ({color = "#000"}: {color?: string}) => {
+  return (
+    <Box height="1px" sx={{backgroundColor: color}} />
+  )
```

Comment:
material also has this? https://mui.com/material-ui/react-divider/

Reply:
same